wom/src/main/scala/wom/graph/DeclarationNode.scala

```diff
@@ -0,0 +1,29 @@
+package wom.graph
+
+import lenthall.validation.ErrorOr.ErrorOr
+import wdl.types.WdlType
+import wom.expression.WomExpression
+import wom.graph.ExpressionNode.buildFromConstructor
+import wom.graph.GraphNodePort.{InputPort, OutputPort}
+
+object DeclarationNode {
```

Comment:
I must have missed this, but can you explain why this has been added now?

Reply:
Yeah I think we could do without that but we would need to change a bit how graph outputs are added to the graph in WOM.
In WDL-land when we create a Graph from a WdlGraphNode, at the end we add GraphOutputNodes to the graph to signify what is to be considered an output of this graph.
We do that by going through all nodes that we've added and creating a `PortBasedGraphOutputNode` if we think it should be an output of the graph. The issue is we want some expression nodes to be outputs, but not all of them. This is to make this distinction